test_conformance/spirv_new/test_spirv_14.cpp

```diff
@@ -0,0 +1,213 @@
+//
+// Copyright (c) 2024 The Khronos Group Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include "testBase.h"
+#include "spirvInfo.hpp"
+#include "types.hpp"
+
+#include <string>
+
+extern bool gVersionSkip;
```

Comment:
Shouldn't be needed, as it's already in `spirvInfo.hpp` (which is the only user)?

Reply:
Correct, fixed.  (I originally had the check function in this file, then I moved it to a separate file so it could be reused.)